standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2121,6 +2159,22 @@ struct AlterPartitionsRequest {
   7: optional string validWriteIdList
 }
 
+struct AppendPartitionRequest {
+  1: optional string catalogName,
+  2: required string dbName,
+  3: required string tableName,
+  4: required string partName,
+  5: optional EnvironmentContext environmentContext
+}
+
+struct AppendPartitionsRequest {
```

Comment:
can this struct be merged with AppendPartitionRequest? most of fields are the same

Reply:
Ack